pkg/pagerduty/pagerduty_test.go

```diff
@@ -284,6 +284,81 @@ var _ = Describe("Pagerduty", func() {
 			})
 		})
 	})
+	Describe("GetAlerts", func() {
```

Comment:
Just as a Note for future work: 
we are testing here only for the error return values, 
there is no testing of an expected actual array of alerts

Reply:
adding these test cases, thanks for pointing it out!